comp/core/tagger/impl/tagger.go

```diff
@@ -538,6 +539,12 @@ func (t *TaggerWrapper) EnrichTags(tb tagset.TagsAccumulator, originInfo taggert
 	}
 }
 
+// GenerateContainerIDFromExternalData generates a container ID from the external data.
+func (t *TaggerWrapper) GenerateContainerIDFromExternalData(externalData origindetection.ExternalData) (string, error) {
```

Comment:
Not sure if this function should be in local_tagger.go instead.

Reply:
Done 👍